src/tools/spin/spinquic.cpp

```diff
@@ -259,8 +355,10 @@ QUIC_STATUS QUIC_API SpinQuicHandleStreamEvent(HQUIC Stream, void * /* Context *
     return QUIC_STATUS_SUCCESS;
 }
 
-QUIC_STATUS QUIC_API SpinQuicHandleConnectionEvent(HQUIC Connection, void * /* Context */, QUIC_CONNECTION_EVENT *Event)
+QUIC_STATUS QUIC_API SpinQuicHandleConnectionEvent(HQUIC Connection, void *Context, QUIC_CONNECTION_EVENT *Event)
 {
+    // TODO: avoid context overwite
+    uint16_t ThreadID = 0; // *(uint16_t*)Context;
```

Comment:
This Context becomes null by (maybe) SetContext called in spin loop.
Fuzz data based GetRandom cannot be used

Reply:
StreamCallback as well